app/openlxp_xds_project/settings.py

```diff
@@ -279,15 +314,5 @@
     "/api/spotlight-courses",
 ]
 
-EMAIL_BACKEND = 'django_ses.SESBackend'
-
-
-# Django-notifications package settings
-DJANGO_NOTIFICATIONS_CONFIG = {
-    'USE_JSONFIELD': True,
-}
-
-# when notifications should be automatically deleted, should be days or greater
-NOTIFICATIONS_EXPIRE_AFTER = datetime.timedelta(days=30)
-
-DATA_UPLOAD_MAX_MEMORY_SIZE = 5242880
+if XAPI_ALLOW_ANON and not XAPI_USE_JWT:
+    OPEN_ENDPOINTS.append("/api/statements")
```

Comment:
Just want to confirm, this still applies the CSRF check, right?

Reply:
Correct, it would just make it so that a specific permission isn't required to hit the API Endpoint

Reply:
Correct, it would just make it so that a specific permission isn't required to hit the API Endpoint